src/main/java/com/google/devtools/build/lib/remote/options/RemoteOptions.java

```diff
@@ -147,6 +147,17 @@ public final class RemoteOptions extends CommonRemoteOptions {
       help = "Whether to fall back to the local downloader if remote downloader fails.")
   public boolean remoteDownloaderLocalFallback;
 
+  @Option(
+      name = "experimental_remote_downloader_propagate_credentials",
+      defaultValue = "false",
+      documentationCategory = OptionDocumentationCategory.REMOTE,
+      effectTags = {OptionEffectTag.UNKNOWN},
+      help =
+          "Whether to propagate credentials from netrc and credential helper to the remote"
+              + " downloader server. This requires the server implementation to support Bazel's"
+              + " `http_header_url` Qualifier.")
```

Comment:
nit: mention the format of this qualifier?

Reply:
To be fair, all the existing bazel-specific qualifiers (canonical id, http_header) are undocumented. Do you think the flag description would be the best place to document this new qualifier? 🤔

Reply:
I will replace the existing sentence with this

```
The server implementation needs to support the new
http_header_url:<url-index>:<header-key> qualifier where the
url-index is a 0-based position of the URL inside the
FetchBlobRequest's uris field.
```

But I think we should find a doc page to document all 3 of these.